dev/cognitiveservices/data-plane/Language/preview/analyzeconversations-authoring.json

```diff
@@ -0,0 +1,3102 @@
+{
```

Comment:
These should be directly under the "Language" directory. No "preview" or "stable". If working on any in parallel, just use branches like teams would normally for parallel development. When ready to merge back, commands like `git merge` and/or `git cherry-pick` just work then, whether merging into or from stable, for example.